content/blog/2021/outreach-partners.md

```diff
@@ -0,0 +1,39 @@
+Title: We are looking for outreach partners for PyCon India 2021
+Date: 2021-04-06 23:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2021
+Slug: outreach-partners
+Summary: We are looking for outreach partners that can help spread the word about our conference in communities all over the world and bring fresh faces to the platform in hopes of creating a more inclusive, accessible conference this year.
+
+[**PyCon India**](https://in.pycon.org/2021/) is looking for outreach partners that can help spread the word about our conference in communities all over the world and bring fresh faces to the platform in hopes of creating a more inclusive, accessible conference this year.
+
+PyCon India is the largest gathering of Python learners, programmers, and enthusiasts in India who come from all over the world once a year to celebrate the programming language and share their experiences, insight, and expertise with the community. This year we like to bring forward positive change, and make the conference even more inclusive by bringing forth combined efforts of our communities in helping PyCon India reach more viewers.
+
+## What we expect from you
+
+We would like to invite your community to be an outreach partner for PyCon India 2021. You can help us spread the word about the conference among your members who are looking to learn more regarding Python. We expect our outreach partners to help us in the following ways:
+
+* Introduce your community to PyCon India 2021 by creating an event on your platform and share the event on your Social Media platforms.
+* Share the content from the conference with your community members on your mailing list, social media channels, and messaging groups.
+* Spread the word about our call for proposals for talks, workshops & dev sprint and help us find speakers, mentors, and reviewers from your community for the conference.
+* Update the PyCon India team after sharing the content from the conference on your platforms so that the team can keep track of our promotions.
+
+## What you can expect from us
+
+For supporting the conference, our outreach partners can expect the following in return from PyCon India:
+
+* We will Acknowledge your community in opening and closing address
+* We will display your community logo on the conference screen between sessions
```

Comment:
let's hold on this?
Next year when we've sponsors, this might conflict.

Reply:
@anistark +1

Reply:
Sure, but this also needs to be removed from the email draft. 
```suggestion
```

Reply:
@abhishekmishragithub Please check the email draft also then. I missed it there.